java/arcs/android/host/AndroidManifestHostRegistry.kt

```diff
@@ -90,7 +90,13 @@ class AndroidManifestHostRegistry private constructor(
     companion object {
         /** Auxiliary constructor with default sender. */
         fun create(ctx: Context) =
-            AndroidManifestHostRegistry(ctx) { intent -> ctx.startService(intent) }.initialize()
+            AndroidManifestHostRegistry(ctx) {
+                intent -> try {
+                    ctx.startService(intent)
+                } catch (e: Exception) {
+                    // TODO: notify Allocator of failure.
```

Comment:
I'm not sure you'll get an informative exception other than a timeout? I haven't written the code yet to parcelize exception messages back through a Bundle in ResultReceiver so that IntentBasedArcHostAdapter can turn it into an exception. You might want to look at that.

Reply:
If this is to merge in interim probably we should at least log a warning?